packages/fast-components-styles-msft/src/text-area/index.ts

```diff
@@ -3,8 +3,8 @@ import { TextAreaClassNameContract } from "@microsoft/fast-components-class-name
 import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
 import { height } from "../utilities/density";
 import { inputFieldStyles } from "../patterns/input-field";
-import { multiply, toPx } from "@microsoft/fast-jss-utilities";
-import { designUnit } from "../utilities/design-system";
+import { format, multiply, toPx } from "@microsoft/fast-jss-utilities";
+import { designUnit, outlineWidth } from "../utilities/design-system";
```

Comment:
looks like you can remove these added imports

Reply:
removed

Reply:
removed